installer/conf/omsagent.d/npmd.conf

```diff
@@ -0,0 +1,14 @@
+<source>
+    type npmd
```

Comment:
The intention is not to deploy these during omsagent install. For this solution to actually start, one would need to enable it in the web portal explicitly (NPMD Solution) and then via DSC we would be copying these over to the respective directories and restarting omsagent.